src/main/java/step4/main/Application.java

```diff
@@ -0,0 +1,27 @@
+package step4.main;
+
+import step4.service.RacingCarGame;
+import step4.domain.UserInput;
+import step4.view.InputView;
+
+import static step4.view.OutputView.printCars;
+import static step4.view.OutputView.printResultConsole;
+import static step4.view.OutputView.printWinners;
+
+public class Application {
+
+    public static void main(String[] args) {
+        final UserInput input = InputView.input();
+        final RacingCarGame game = new RacingCarGame(input.getName());
+
+        printResultConsole();
+
+        for (int i = 0; i < input.getRetry(); i++) {
+            game.playing();
+
+            printCars(game);
+        }
```

Comment:
일종의 컨트롤러에서 비즈니스로직을 수행하고 있는 것으로 보입니다. 
이런식으로 바꿔보는건 어떨까요? 
```suggestion
       GameResult result = game.play();
       printResult(result);
```